manuscript/20.supp-figures.md

```diff
@@ -3,40 +3,40 @@
+![**Performance on censored multiple lineages of increasing size.** (a) Visualization of cells increasing in size and number of censored lineages. (b) Bernoulli parameter approaches the true value for state 1 as the number of cells increase and approaches the true value between 100 to 200 cells and deviates for larger lineages for state 2. (c) Gamma shape parameter approaches the true value as the number of cells increase for states 1 and 2. (d) Gamma scale parameter approaches the true value consistently for state 2 and as the number of cells increase for state 1. (e) State assignment accuracy approaches 100% as the number of cells increase. (f) Errors in estimating the transition probability matrix, T, approaches 0 as the number of cells increase. (g) Errors in estimating the initial probability matrix, π, approaches 0 as the number of cells increase.](./output/figureS03.svg){#fig:cenMulti}
 
 \pagebreak
 
 <!-- # (Supp. Figure 4) -->
 ### Change in model performance when varying presence of a state for uncensored population
 
-![**Model performance relative to the presence of each state in uncensored lineages.** (a) Visualization of the distribution of cells in the lineage transitioning from state 1 to state 2. (b) Bernoulli parameter approaches the true value when 40-80% of cells are in state 1 and deviates at other percentages for states 1 and 2. (c) Gamma shape parameter approaches the true value then deviates when over 60% of cells are in state 1 for states 1 and 2. (d) Gamma scale parameter approaches the true value consistently for states 1 and 2. (e) State assignment accuracy decreases as the percent of cells in State 1 increases. (f) Errors in estimating the transition probability matrix, T, approaches 0 when 40-75% of cells are in state 1 and increases at other percentages for states 1 and 2.](./output/figureS04.svg){#fig:prop4}
+![**Model performance relative to the presence of each state in uncensored lineages.** (a) Visualization of the distribution of cells in the lineage transitioning from state 1 to state 2. (b) Bernoulli parameter approaches the true value as the number of cells increase for states 1 and 2 and deviates from the true value for state 2 when 60% of cells are in state 1. (c)-(d) Gamma shape and scale parameters approach the true value when number of cells increase for states 1 and 2. (e) State assignment accuracy approaches 100% consistently as the number of cells in state 1 increase. (f) Errors in estimating the transition probability matrix, T, approaches 0 consistently.](./output/figureS04.svg){#fig:prop4}
```

Comment:
The number of cells doesn't change here. The proportion of cells in state 1 and 2 changes.